modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -259,8 +364,8 @@ def __run(self, cmd):
 
         return output
 
-    def __form_cmd(self, node, cmd):
-        return self._cluster.spec.path.script(f"{self.BASE_COMMAND} --host {node.account.externally_routable_ip} {cmd}")
+    def __form_cmd(self, cmd):
+        return self._cluster.spec.path.script(f"{self.BASE_COMMAND} --host `hostname -i` {cmd}")
```

Comment:
from man "Display the IP address**(es)** of the host". So is it ok that there could be multiple IP addresses  for the host param?

Reply:
fixed